cmd/gateway-main.go

```diff
@@ -209,6 +209,7 @@ func gatewayMain(ctx *cli.Context) {
 	endpointAddr := ctx.Args().Get(1)
 	// Third argument is the address flag
 	serverAddr := ctx.String("address")
+	fatalIf(CheckLocalServerAddr(serverAddr), "Invalid address ‘%s’ in command line argument.", serverAddr)
```

Comment:
The better question to ask is is this a show stopper fix for this release. GCS is the next after this one..so  it doesn't make sense to duplicate a fix. If needed IMO the original fix sent to GCS should be sent here by the same submitter.